### PR TITLE
issue#277 Alert updated

### PR DESCRIPTION
**Pull request by isha-73**
Error alert has been updated!

### CODE REVIEW

File: src/pages/Login.jsx [hunk 1]
```diff
@@ -52,11 +56,11 @@ const LoginScreen = () => {
         });
       })
       .catch((error) =>
-        enqueueSnackbar(error.message, {
+        enqueueSnackbar("Something went wrong. Make sure you already have an account linked with this email-id!", {
```
Comment (narayan954) [inline src/pages/Login.jsx:59]: We can't hardcode this error message, someone without an account can also directly login using google sign in.(handled by firebase)

File: src/pages/Login.jsx [hunk 2]
```diff
@@ -69,18 +73,19 @@ const LoginScreen = () => {
         });
       })
       .catch((error) =>
-        enqueueSnackbar(error.message, {
+        enqueueSnackbar("Make sure your account is linked to your facebook account", {
```
Comment (narayan954) [inline src/pages/Login.jsx:76]: We can't hardcode this error message, there're several error messages left to be handled. For example : Firebase: An account already exists with the same email address but different sign-in credentials. Sign in using a provider associated with this email address. (auth/account-exists-with-different-credential).

File: src/pages/Login.jsx [hunk 3]
```diff
@@ -10,12 +10,14 @@ import { faRightToBracket } from "@fortawesome/free-solid-svg-icons";
...
 import { useSnackbar } from "notistack";
 
+
 const LoginScreen = () => {
   const classes = useStyles();
 
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
   const [showPassword, setShowPassword] = useState(false);
+  const [error, setError] = useState(null);
```
Comment (narayan954) [inline src/pages/Login.jsx:20]: it's not used

File: src/pages/Login.jsx [hunk 4]
```diff
@@ -34,14 +36,16 @@ const LoginScreen = () => {
           variant: "success",
         });
       })
-      .catch((error) =>
-        enqueueSnackbar(error.message, {
+      .catch((error) => {
+        enqueueSnackbar("Please try again with correct information!", {
```
Comment (narayan954) [inline src/pages/Login.jsx:40]: correct credentials suits better right or is it fine?

**Review by narayan954 — CHANGES_REQUESTED** [review]

**Comment by narayan954** [comment]
> Error alert has been updated!

Which issue is this?? Please create an issue to link to a pull request if it doesn't exists

**Comment by narayan954** [comment]
Also, for this issue, I'd suggest you to go through the types of errors Firebase generates for those functions, so you can come to a concluding generic message easily.

File: src/pages/Login.jsx (hunk 1 above)
Comment (isha-73) [reply src/pages/Login.jsx:59]: So, should I keep it as it was earlier?

File: src/pages/Login.jsx (hunk 3 above)
Comment (isha-73) [reply src/pages/Login.jsx:20]: Yeah, will remove it

**Comment by isha-73** [comment]
> Also, for this issue, I'd suggest you to go through the types of errors Firebase generates for those functions, so you can come to a concluding generic message easily.

Okay fine

File: src/pages/Login.jsx (hunk 1 above)
Comment (narayan954) [reply src/pages/Login.jsx:59]: > So, should I keep it as it was earlier?

You can go through the Firebase documentation to see what actual errors this method raise and decide

**Comment by narayan954** [comment]
> > > Error alert has been updated!
> > 
> > 
> > Which issue is this?? Please create an issue to link to a pull request if it doesn't exists
> 
> It is issue 277

Issue 277 is about client side validation :/, this is handling the server response. We can validate on the client side itself for email formatting...

**Comment by isha-73** [comment]
@narayan954 can u check now, as u told I have updated alerts based on firebase errors.

**Review by narayan954 — APPROVED** [review]
looks good to me!

**Pull request merged**